docs/core/additional-tools/dotnet-core-uninstall.md

```diff
@@ -0,0 +1,357 @@
+---
+title: .NET Core Uninstall Tool
+description: An overview of the .NET Core Uninstall Tool, a guided tool that enables the controlled clean-up of .NET Core SDKs and runtimes.
+author: yuchong-pan
+ms.date: 10/31/2019
+---
+# .NET Core Uninstall Tool
+
+The [.NET Core Uninstall Tool](https://github.com/dotnet/cli-lab/releases) (`dotnet-core-uninstall`) lets you clean up .NET Core SDKs and runtimes on a system such that only the desired versions remain. A collection of options is available to specify which versions you want to uninstall.
+
+The tool supports Windows and macOS. Linux is currently not supported.
+
+Note that the Windows version of this tool can only uninstall .NET Core SDKs and runtimes that were installed using .NET Core SDK and runtime installers. Prior to Visual Studio 2019 Update 16.3, the Visual Studio installer called the .NET Core SDK installer and these versions can be uninstalled with the tool. The `dotnet-core-uninstall list` command displays which SDKs can be uninstalled with the tool.
```

Comment:
What happens starting with 16.3?

Reply:
@wli3 would know more than me

Reply:
@wli3 can you please comment?

Reply:
After that Visual Studio is in charge of its own copy. That means Visual Studio will bring a copy of dotnet core, and the user cannot uninstall it from "add remove programs". And Visual Studio will also update that copy in VS update. It will also go away when VS is uninstalled